tests/apps/tests.py

```diff
@@ -367,6 +367,10 @@ def test_duplicate_dunder_path_no_dunder_file(self):
         ac = AppConfig('label', Stub(__path__=['a', 'a']))
         self.assertEqual(ac.path, 'a')
 
+    def test_repr(self):
+        app = AppConfig('label', Stub(__path__=['a']))
+        self.assertEqual(app.__repr__(), '<AppConfig: label>')
```

Comment:
The more typical way to test `__repr__` or `__str__` is to use `repr(app)` or `str(app)`.